### PR TITLE
Adding skeleton file for City of LA

### LINKED ISSUE #19: mmoralJPL adds City of Los Angeles File
This is a tracking issue about the city of Los Angeles.

Things to do:
- [ ] Places to eat
- [ ] Things to do

### PR DESCRIPTION
**Pull request by mmoralJPL**
File is for things to do in LA.

@amahpour What do you think? :+1: 

This pull request fixes #19

### CODE REVIEW

File: city-of-la.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+City of LA
```
Comment (amahpour) [inline city-of-la.md:1]: I like LA 👍

File: city-of-la.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (alortega) [inline city-of-la.md]: Hi! You may want to consider having subsections for different neighborhoods of the city.

File: city-of-la.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+Places to eat
```
Comment (amahpour) [inline city-of-la.md:6]: 🍔

**Review by amahpour — COMMENTED** [review]
I like it!

**Review by alortega — COMMENTED** [review]
Great job!

File: city-of-la.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+City of LA
+  Subsection of LA that I am familiar with are:
+  - Highland Park
+  - Eagle Rock
+  
+Places to eat
+
```
Comment (amahpour) [inline city-of-la.md]: I see nothing here.

**Review by amahpour — CHANGES_REQUESTED** [review]
Requesting @mmoralJPL to add more content

**Review by amahpour — APPROVED** [review]

**Pull request merged**